### DISCUSSION TITLE
Save Record Confirmation | Include name of record saved

### DISCUSSION THREAD

**Discussion by HunterJacobs** [burkybang/SuiteAdvanced-Feedback#136] [Ideas]
Would it be possible to have the name or the record that was just saved in the below popup? 
![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/6316084/3d993932-5108-485d-84fa-19fe292b2b87)

**Comment by burkybang** [comment]
Where and when is this message shown?

**Reply by HunterJacobs** [reply]
This message shows up when I save a email template, have not looked/noticed it anywhere else
